topics/admin/tutorials/backup-cleanup/tutorial.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: tutorial_hands_on
+
+title: "Server Maintenance: Backups & Cleanup"
+questions:
+- How can I back up my Galaxy?
+- What data should be included?
+- How can I ensure jobs get cleaned up appropriately?
+- How do I maintain a Galaxy server?
+- What happens if I lose everything?
+objectives:
+- Learn about different maintenance steps
+- Setup postgres backups
+- Setup cleanups
+- Learn what to back up and how to recover
+time_estimation: "30m"
+key_points:
+  - Use configuration management (e.g. Ansible)
+  - Store configuration management in git
+  - Back up the parts of Galaxy that can't be recreated
+contributions:
+  authorship:
+  - hexylena
+  - natefoo
```

Comment:
@natefoo can you look over this one briefly?

We dont' discuss restoration, but I know you had documentation for that in a PR, could you link that here and I'll write a small section on restoration?

Reply:
Here is the link: https://github.com/galaxyproject/ansible-postgresql/pull/30#issuecomment-963600656
And here is how I struggled but managed to do it:
https://github.com/lldelisle/galaxyduboule-infrastructure/blob/778e5b056f03970a1af5fc2f9dd133b4e4791cac/testUpgradeOnVM.sh#L53-L130

Reply:
oh!!! amazing. I owe you a drink next time we meet :) this saves so much time

Reply:
You see I am a good beta-tester... :wink: